components/list/examples/and_list.rs

```diff
@@ -6,12 +6,15 @@
 icu_benchmark_macros::instrument!();
 use icu_benchmark_macros::println;
 
-use icu::list::{ListFormatter, ListLength};
+use icu::list::{ListFormatter, ListFormatterOptions, ListLength};
 use icu::locale::locale;
 
 fn main() {
-    let list_formatter =
-        ListFormatter::try_new_and_with_length(&locale!("es").into(), ListLength::Wide).unwrap();
+    let list_formatter = ListFormatter::try_new_and_with_length(
+        locale!("es").into(),
+        ListFormatterOptions::new().style(ListLength::Wide),
```

Comment:
I think we don't need a `new` if we have a `default`. Do we have a `default`?

Reply:
We do, I think having `new` and `default` is common and I prefer it.

Reply:
I don't think `new` is all that common on options bags (struct with only public optional fields). The `Default` implementation is clear, it returns a struct with all fields set to `None` (and that's why we should use `default()` in docs). This is not the expectation for `new`, so I think it adds friction to the API. It also adds another way of doing the same thing, so it makes people question if it actually does the same thing.

Reply:
I'm convinced. thank you!